app/partials/group_management_details.html

```diff
@@ -38,6 +38,11 @@ <h6>Invite students<span class="show-for-medium-up"> to group</span> via email</
                             </td>
                             <td ng-if="!member.authorisedFullAccess"><span class="group-table-person"></span><span class="revoked">{{member.givenName}} {{member.familyName}}</span> (not sharing)</span></td>
                             <td ng-if="false" class="group-table-edit"><a class="edit-icon show-for-small-only" href="javascript:void(0)"></a></td>
+                            <td>
+                                <a href="javascript:void(0)" ng-click="resetMemberPassword(member)" class="right" ng-class="{disabled: !canSendPasswordResetRequest(member)}"
+                                aria-haspopup="true" data-ot="{{canSendPasswordResetRequest(member) ? 'Send a password reset request to your student\'s email address.' : member.passwordRequestSent ? 'Password reset request sent.' : 'Cannot send password reset request email.'}}">
```

Comment:
I'm not sure these messages are clear enough. We don't want teachers misunderstanding things.

Maybe we can have a greyed out "Reset email sent" when it's disabled because we sent it, mention the student's invalid email when that's the reason we can't send it, and just not show it when they're not sharing their data?